### PR TITLE
Omnibus package build for openresty

### PR DESCRIPTION
**Pull request by bearnard**
- Builds ubuntu packages for nginx openresty and all it's dependencies
  parent: https://github.com/yola/p-p-platform/issues/49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,107 +1,65 @@
...
+$ vagrant plugin install vagrant-omnibus vagrant-vbguest vagrant-berkshelf
 ```
 
-Adding the `--purge` purge option removes __ALL__ files generated during the
-build including the project install directory (`/opt/nginx-openresty`) and
-the package cache directory (`/var/cache/omnibus/pkg`):
-
 ```shell
-$ bin/omnibus clean --purge
+$ sudo gem install bundler && sudo gem update bundler
```
Comment (adrianmoisey) [inline README.md]: This doesn't fit with the assumption on line 8

File: README.md [hunk 2]
```diff
@@ -1,107 +1,65 @@
...
-`nginx-openresty`!
-
-[![Build Status](https://travis-ci.org/bearnard/omnibus-nginx-openresty.png)](https://travis-ci.org/bearnard/omnibus-nginx-openresty.png)
+This project creates full-stack ubuntu platform-specific packages for
+`openresty - http://openresty.org`!
 
 ## Installation
 
 We'll assume you have Ruby 1.9+ and Bundler installed. First ensure all
 required gems are installed and ready to use:
```
Comment (adrianmoisey) [inline README.md]: Last sentence here seems out of place

File: README.md [hunk 3]
```diff
@@ -1,107 +1,65 @@
...
+bundle exec kitchen login openresty-$RELEASE
+...
+cd omnibus-nginx-openresty
+bundle install --without development # Don't install dev tools!
+bundle exec omnibus build openresty --override append_timestamp:false
 
-If you would like to build a package for a single platform the command looks like this:
+```
 
+Then logout of the instance to upload the resultant deb in ./pkg to an s3 repo. run:
```
Comment (adrianmoisey) [inline README.md:42]: This text is inside of a quoted block

File: config/software/openresty.rb [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+name "openresty"
+default_version "1.7.10.2"
+
+dependency "pcre"
+dependency "openssl"
+dependency "zlib"
+
+version("1.7.10.2") { source md5: "bca1744196acfb9e986f1fdbee92641e" }
+version("1.4.3.6") { source md5: "5e5359ae3f1b8db4046b358d84fabbc8" }
```
Comment (adrianmoisey) [inline config/software/openresty.rb]: Is this version used?

File: README.md [hunk 5]
```diff
@@ -1,107 +1,65 @@
...
-platforms with the following command:
+Then login to the instance and build the project:
 
 ```shell
-$ vagrant up
-```
+bundle exec kitchen login openresty-$RELEASE
+...
+cd omnibus-nginx-openresty
+bundle install --without development # Don't install dev tools!
```
Comment (adrianmoisey) [inline README.md]: This comment doesn't give any more info then the command itself. Remove it or make it say why we don't need the dev tools?

File: build.sh [hunk 6]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (adrianmoisey) [inline build.sh:1]: This script looks like it does what is in the readme. Why not just reference it from the readme?

File: config/software/openresty.rb [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    '--without-mail_pop3_module',
+    '--with-ipv6',
+    # AIO support define in Openresty cookbook. Requires Kernel >= 2.6.22
+    # Ubuntu 10.04 reports: 2.6.32-38-server #83-Ubuntu SMP
+    # However, they require libatomic-ops-dev and libaio
+    #'--with-file-aio',
+    #'--with-libatomic'
+  ]
+
+  # OpenResty 1.7 + RHEL5 Fixes:
```
Comment (adrianmoisey) [inline config/software/openresty.rb]: Do we care about RHEL?

File: config/software/openresty.rb (hunk 4 above)
Comment (bearnard) [inline config/software/openresty.rb]: Not by us, happy to remove it.

File: config/software/openresty.rb (hunk 7 above)
Comment (bearnard) [inline config/software/openresty.rb]: We don't

File: build.sh (hunk 6 above)
Comment (bearnard) [inline build.sh:1]: This snuck in, I'll update it and reference it.

File: README.md (hunk 5 above)
Comment (bearnard) [inline README.md]: sure.

**Comment by bearnard** [comment]
:bell:

**Comment by stormf** [comment]
LGTM :+1:

**Comment by stormf** [comment]
:+1:

**Pull request merged**